de_DE/webapp/src/main/webapp/templates/freemarker/visualization/capabilitymap/capabilityMap_de_DE.ftl

```diff
@@ -15,8 +15,13 @@ ${stylesheets.add(
 )}
 
 <script language="JavaScript" type="text/javascript">
+    var i18nStringsCap = {
+        term: '${i18n().group_capitalized}',
+        group: '${i18n().term_capitalized}'
+    };
     var contextPath = "${urls.base}";
     $(document).ready(function() {
+        document.title = "Capability Map";
```

Comment:
Can we instead use an i18n property? instead of hardcoding the language-specific value here?

Reply:
okay, i added this and added an i18n property for the title